rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/41_date_range.yml

```diff
@@ -0,0 +1,178 @@
+setup:
+  - do:
+      indices.create:
+          index: test
+          body:
+            settings:
+              number_of_replicas: 0
+            mappings:
+              properties:
+                date:
+                  type: date
+                  format: epoch_second
+
+  - do:
+      cluster.health:
+        wait_for_status: green
```

Comment:
We actually wait for yellow by default which is enough. I think these are mostly left over from the before times.

Reply:
yeah, again, I just copied what the existing test did.